cve_bin_tool/checkers/bluez.py

```diff
@@ -23,5 +23,4 @@ def get_version(lines, filename):
     if "is_or_contains" in version_info:
         version_info["modulename"] = "bluetoothctl"
         version_info["version"] = regex_find(lines, *regex)
-
```

Comment:
Housekeeping:  This is an unecessary change, so we might as well leave it out.  It's considered bad form in a lot of projects to leave unnecessary whitespace changes in a PR (necessary ones would be the ones required by Black in our case, or by your style guide for other projects).  We're supposed to encourage good habits in GSoC so please fix this before we merge!

Reply:
@terriko This change was done by the Black formatter. Although in gnutls.py it was my mistake. I'll take care of it in the future